From 97a58af643843696c7ca435c60f3dba7eb47b7fb Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
Date: Fri, 8 Jan 2021 15:14:49 +0200
Subject: [PATCH 37/78] clocksource/drivers/fsl_stm: Rename the file for
 consistency

For the sake of consistency, let's rename the file to a name similar
to other file names in this directory.

Issue: ALB-5454
Signed-off-by: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
---
 drivers/clocksource/Makefile                             | 2 +-
 drivers/clocksource/{fsl_stm_timer.c => timer-fsl-stm.c} | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename drivers/clocksource/{fsl_stm_timer.c => timer-fsl-stm.c} (100%)

diff --git a/drivers/clocksource/Makefile b/drivers/clocksource/Makefile
index c89923ab2066..096bf8bf1b4f 100644
--- a/drivers/clocksource/Makefile
+++ b/drivers/clocksource/Makefile
@@ -49,7 +49,7 @@ obj-$(CONFIG_CLKSRC_LPC32XX)	+= timer-lpc32xx.o
 obj-$(CONFIG_CLKSRC_MPS2)	+= mps2-timer.o
 obj-$(CONFIG_CLKSRC_SAMSUNG_PWM)	+= samsung_pwm_timer.o
 obj-$(CONFIG_FSL_FTM_TIMER)	+= timer-fsl-ftm.o
-obj-$(CONFIG_FSL_STM_TIMER)     += fsl_stm_timer.o
+obj-$(CONFIG_FSL_STM_TIMER)     += timer-fsl-stm.o 
 obj-$(CONFIG_VF_PIT_TIMER)	+= timer-vf-pit.o
 obj-$(CONFIG_FSL_GLOBAL_TIME_STM) += fsl_global_time.o
 obj-$(CONFIG_FSL_STM_TIMER)	+= fsl_stm_timer.o
diff --git a/drivers/clocksource/fsl_stm_timer.c b/drivers/clocksource/timer-fsl-stm.c
similarity index 100%
rename from drivers/clocksource/fsl_stm_timer.c
rename to drivers/clocksource/timer-fsl-stm.c
-- 
2.25.1

